src/components/app/managers/KeyboardShortcutsManager.tsx

```diff
@@ -194,24 +201,49 @@ const KeyboardShortcutsManager: FC = () => {
             ],
             [
                 "shift+ArrowDown",
-                () =>
+                (event) =>
+                    !event.repeat &&
                     typeof localVolume.value !== "undefined" &&
                     localVolume.value > 0 &&
                     setLocalVolume({
                         value: Math.max(localVolume.value - largeVolumeDelta, 0),
                         whenSet: Date.now(),
                     }),
             ],
-            ["ctrl+shift+ArrowDown", () => amplifierMuteToggle()],
-        ];
-    }, [amplifierMuteToggle, localVolume, volumeLimit]);
+        ],
+        [localVolume, volumeLimit, setLocalVolume]
```

Comment:
I'm not sure exactly what I'm doing with these deps lists - could you please review these carefully? Thanks.

Reply:
Thanks for taking the time to type this up! Yes, indeed, I was wondering why the amplifier mutations were included in the deps list, but not the mutator for these local state things. Your explanation has helped, and I've removed `setLocalVolume` from this list.

Reply:
Great! Much appreciated.